### PR TITLE
Us103 anna

### PR DESCRIPTION
**Pull request by AnnaLoshak**
please check my code

### CODE REVIEW

File: src/us100/us103/Anna.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package us100.us103;
+
+public class Anna {
+
```
Comment (BadamkhandB) [inline src/us100/us103/Anna.java:4]: Anna you did greeat job

**Review by tsjuder1 — APPROVED** [review]
everything looks great

File: src/us100/us103/Anna.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    //work 1
+
+    //work2
+}
```
Comment (VeronikaKrasnokutska) [inline src/us100/us103/Anna.java:8]: complete work3, please

**Review by VeronikaKrasnokutska — CHANGES_REQUESTED** [review]

**Review by VeronikaKrasnokutska — APPROVED** [review]
Good job

**Pull request merged**